pages/dkp/kommander/1.3/projects/platform-services/platform-services-catalog/kudo-cassandra/index.md

```diff
@@ -117,6 +117,9 @@ KUDO Cassandra versions can be upgraded using the KUDO CLI.
 
 **Example** Upgrade KUDO Cassandra from `v3.11.6-1.0.1` to `v3.11.7-1.0.2`:
 When upgrading, you should understand the mapping between Cassandra versions and operator versions.   For more information, see the table at the end of the [Cassandra operator](https://github.com/mesosphere/kudo-cassandra-operator/tree/master/docs)
+```
```

Comment:
Style point - but do we want an extra break/enter here between the code snippet and the paragraph?
cc: @russurquhart1 @JohnB64-JBV

Reply:
nah, we have spacing in the CSS that handles this.